packages/lake/src/components/Accordion.tsx

```diff
@@ -42,9 +43,10 @@ const styles = StyleSheet.create({
 type Props = {
   trigger: ReactNode;
   children: ReactNode;
+  withSeparator?: boolean;
```

Comment:
I don't think we need this prop because we can add add `<Separator />` between 2 `<Accordion>` but maybe I miss something